src/llm/apis/openai_completions.cpp

```diff
@@ -167,28 +184,7 @@ absl::Status OpenAIChatCompletionsHandler::parseMessages() {
                             if (!absl::Base64Unescape(std::string_view(url.data() + offset, url.size() - offset), &decoded)) {
                                 return absl::InvalidArgumentError("Invalid base64 string in request");
                             }
-                            int rows = 1;
-                            int cols = decoded.size();
-                            cv::Mat rawData(rows, cols, CV_8UC1, (void*)decoded.data());
-                            cv::Mat image;
-                            try {
-                                image = cv::imdecode(rawData, cv::IMREAD_UNCHANGED);
-                            } catch (const cv::Exception&) {
-                                return absl::InvalidArgumentError("Error during string to mat conversion");
-                            }
-                            std::vector<size_t> shape;
-                            shape.push_back(image.rows);
-                            shape.push_back(image.cols);
-                            shape.push_back(image.channels());
-                            auto type = getOvTypeFromMatType(image.depth());
-                            if (type == ov::element::dynamic) {
-                                return absl::InvalidArgumentError("Image type is invalid");
-                            }
-                            ov::Tensor tensor(type, shape);
-                            if (image.total() * image.elemSize() != tensor.get_size()) {
-                                return absl::InvalidArgumentError("Image size invalid");
-                            }
-                            memcpy((char*)tensor.data(), (char*)image.data, image.total() * image.elemSize());
+                            ov::Tensor tensor = load_image_stbi(decoded);
```

Comment:
Do we make a copy of underlaying data layer here? Or is it more like an ownership transfer?